docs/integrations/saas-cloud/microsoft-exchange-tracelogs.md

```diff
@@ -0,0 +1,134 @@
+     "Status": "Delivered",
+     "ToIP": null,
+     "FromIP": "20.97.34.221",
+     "Size": 145153,
+     "MessageTraceId": "aa1a3e28-8967-4bfb-06d3-08dad1cb7a0e",
+     "StartDate": "2022-11-27T17:08:26.8492706Z",
+     "EndDate": "2022-11-29T17:08:26.8492706Z",
+     "Index": 0
+   },
+   {
+     "Organization": "org.onmicrosoft.com",
+     "MessageId": "<c41de266-153b-487c-a142-618d99505b38@az.southcentralus.unknown.microsoft.com>",
+     "Received": "2022-11-29T05:35:07.7450561",
+     "SenderAddress": "azure-noreply@microsoft.com",
+     "RecipientAddress": "lmilby@org.onmicrosoft.com",
+     "Subject": "Azure AD Identity Protection Weekly Digest",
+     "Status": "Delivered",
+     "ToIP": null,
+     "FromIP": "20.97.34.221",
+     "Size": 144900,
+     "MessageTraceId": "0c1d5652-d2eb-496c-a741-08dad1cb79be",
+     "StartDate": "2022-11-27T17:08:26.8492706Z",
+     "EndDate": "2022-11-29T17:08:26.8492706Z",
+     "Index": 1
+   },
+   {
+     "Organization": "org.onmicrosoft.com",
+     "MessageId": "<072e0c5d-66d0-40ac-a376-52b11344255a@az.northcentralus.unknown.microsoft.com>",
+     "Received": "2022-11-29T05:35:07.2828271",
+     "SenderAddress": "azure-noreply@microsoft.com",
+     "RecipientAddress": "sumo-apps@org.onmicrosoft.com",
+     "Subject": "Azure AD Identity Protection Weekly Digest",
+     "Status": "Delivered",
+     "ToIP": null,
+     "FromIP": "20.51.6.33",
+     "Size": 145123,
+     "MessageTraceId": "4d68a1db-9375-4ae8-1f6f-08dad1cb7978",
+     "StartDate": "2022-11-27T17:08:26.8492706Z",
+     "EndDate": "2022-11-29T17:08:26.8492706Z",
+     "Index": 2
+   }
+ ]
+}
+```
+
+### Sample Queries​
+
+```sql title="Message Delivery Status"
+_sourceCategory="o365dummy"
+| json "Organization", "MessageId", "Received","SenderAddress", "RecipientAddress", "Subject", "Status", "ToIP", "FromIP", "Size", "MessageTraceId", "StartDate", "EndDate", "Index" as organization, message_id, received, sender_address, recipient_address, subject, status, toIP, fromIP, size, message_traceId, start_date, end_Date, index nodrop
+| where status matches"{{status}}"
+| where sender_address matches"{{sender_address}}"
+| where organization matches "{{organization}}"
+| lookup latitude, longitude, country_code, country_name, region, city, postal_code from geo://location on ip = fromIP
+| where country_name matches "{{country}}"
+| timeslice 1d
+| count by _timeslice, status
+| fillmissing timeslice, values all in status
+| transpose row _timeslice column status
+```
+
+## Collecting logs from Microsoft Exchange Tracelogs App
+
+This section provides instructions for setting up [Cloud-to-Cloud-Integration for Microsoft Exchange Tracelogs App](https://help.sumologic.com/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-exchange-tracelogs/) to create the source and use the same source category while installing the app.
+
+## Installing the Gmail Trace Logs app
```

Comment:
Microsoft, not Gmail

Reply:
Thank you so much for pointing out this.